src/guides/v2.2/ui_comp_guide/components/ui-checkbox-toggle-notice.md

```diff
@@ -0,0 +1,68 @@
+---
+group: ui-components-guide
+title: CheckboxToggleNotice component
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The CheckboxToggleNotice component implements the [`Checkbox`]({{ page.baseurl }}/ui_comp_guide/components/ui-checkbox.html) component with the ability to switch notice according to the selected option.
+
+## Configuration options
+
+| Option | Description | Type | Default |
+| --- | --- | --- | --- |
+| `component` | The path to the component’s JS constructor in terms of RequireJS. | String | `Magento_Ui/js/form/element/single-checkbox-toggle-notice` |
+| `notices` | The array of notices that should be associated by key with selected option value. | Array | `[]` |
+| `tracks`.`notice` | Flags that enable tracking for the `notice` option. | Boolean | `true` |
```

Comment:
Hi @serhiyzhovnir. Could you provide a bit more information information about this option? With the current explanation, a reader might still have questions regarding purpose of this option. 

Thank you!